### PR TITLE
Melee weapon agony damage adjustment and stun baton fix

### PR DESCRIPTION
**Pull request by Jackofashepard**
Power gloves now act more as a assist in combat rather than a 2 click win due to reduced agony making stun batons a preferable weapon for melee non-lethal takedowns (also removed the insta stun on these as well as reduced agony on the cattle prod)

### CODE REVIEW

File: code/modules/clothing/gloves/stungloves.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	action_button_name = "Toggle Power Glove"
 	price_tag = 500
 	var/stunforce = 0
-	var/agonyforce = 50
+	var/agonyforce = 20
```
Comment (SpadesSlick) [inline code/modules/clothing/gloves/stungloves.dm]: This is just plain bad, I will need 5 hits with gloves to down someone. Make it 30 at least.

Comment (Jackofashepard) [reply code/modules/clothing/gloves/stungloves.dm]: Done

**Comment by SpadesSlick** [comment]
Title PR is misleading now.

**Comment by Jackofashepard** [comment]
True let me change it

**Comment by Jackofashepard** [comment]
Done

**Pull request merged**